src/utils/useNumber.ts

```diff
@@ -16,3 +16,33 @@ export const useNumberFormatter = (): any => {
 
   return useMemo(() => formatter, []);
 };
+
+/**
+ * Parser
+ * ref: https://observablehq.com/@mbostock/localized-number-parsing
+ */
+export const useNumberParser = (locale?: string): any => {
```

Comment:
I added this because when I did a really big outplanting, 5000, to test something, the way we were usually converting the value using `+` was failing on `5,000`

Reply:
Oh, just seeing that I should have employed `getLocaleToUse` here. I'll make that update

Reply:
yes, we used to have this parser approach for nursery withdrawals.. we removed it after Steve added `(raw)` fields in search responses.. returns non-localized values.. we use them in most withdrawals math

Reply:
I could have put `useLocalization` inside here instead of forcing it to be passed in. I'll make that change

Reply:
see PR when we got rid of parsing https://github.com/terraware/terraware-web/pull/1196/files

Reply:
Oh heck, I didn't know about that... I'll look into using `raw` instead. Thanks, Karthik

Reply:
no worries, i can relate to this issue you ran into.. a lot 🤣